.babelrc

```diff
@@ -0,0 +1,6 @@
+// .babelrc
+{
+  "presets": [
+    ["es2015"]
```

Comment:
Maybe in a differnt PR when we review browser matrix but `babel-preset-env` instead of es2015 with the browser support?